datasets/jrc-gsw/jrc-gsw-example.ipynb

```diff
@@ -0,0 +1,270 @@
+    "              24.463400705082282\n",
+    "            ],\n",
+    "            [\n",
+    "              91.50238037109375,\n",
+    "              24.71315296906617\n",
+    "            ],\n",
+    "            [\n",
+    "              90.9393310546875,\n",
+    "              24.835349273134295\n",
+    "            ],\n",
+    "            [\n",
+    "              90.81161499023438,\n",
+    "              24.648265332632818\n",
+    "            ],\n",
+    "            [\n",
+    "              90.89813232421875,\n",
+    "              24.337086982410497\n",
+    "            ],\n",
+    "            [\n",
+    "              90.90774536132812,\n",
+    "              24.11792837933617\n",
+    "            ],\n",
+    "            [\n",
+    "              91.07803344726562,\n",
+    "              24.082828048468436\n",
+    "            ]\n",
+    "          ]\n",
+    "        ]\n",
+    "      }"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "We can now execute a STAC API query for our selected area:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Returned 1 Items\n"
+     ]
+    }
+   ],
+   "source": [
+    "catalog = Client.open(\"https://planetarycomputer-staging.microsoft.com/api/stac/v1\")\n",
+    "jrc = catalog.search(collections=[\"jrc-gsw\"], intersects=area_of_interest)\n",
+    "\n",
+    "items = list(jrc.get_items())\n",
+    "\n",
+    "print(f\"Returned {len(items)} Items\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "A single item is returned, so we can work with single assets. If we chose an area with multiple tiles intersecting, we could save use something like [stacstack](https://github.com/gjoseph92/stackstac) or [gdal.BuildVRT](https://gdal.org/python/osgeo.gdal-module.html#BuildVRT) to work with multiple items and assets as a single layer."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 4,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "item = items[0]"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Let's capture the color maps that are included in the COG assets by reading the metadata of the GeoTIFFs using rasterio. This also converts the color maps into a format matplotlib understands:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 8,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "cog_assets = [asset_key for asset_key, asset in item.assets.items() if asset.media_type == pystac.MediaType.COG]"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 12,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "cmaps = {}\n",
+    "for asset_key in cog_assets:\n",
+    "    asset = item.assets[asset_key]\n",
+    "    if asset_key in cog_assets:        \n",
```

Comment:
This `if` condition could be removed since it should always be true?

Reply:
yep, leftover from a different way of looping... fixed